src/main.js

```diff
@@ -93,12 +90,17 @@ if (
 
 app.component("inline-svg", InlineSvg);
 app.use(i18n);
-app.use(PrimeVue, { ripple: true });
+app.use(VueTippy, {
+  // https://vue-tippy.netlify.app/installation
+  directive: "tooltip", // => use as <button v-tippy="your text"></button>
+  defaultProps: {
+    placement: "auto",
```

Comment:
<img width="522" alt="Screenshot 2021-11-27 at 3 47 59 AM" src="https://user-images.githubusercontent.com/13185699/143658538-3436614f-0eea-4494-a0f9-c7149348e91d.png">
is `auto` the right choice? for example, this doesn't look that good to me in terms of positioning.